doc/source/api/process_lock.rst

```diff
@@ -17,3 +17,18 @@ Decorators
 ----------
 
 .. autofunction:: fasteners.process_lock.interprocess_locked
+
+.. code-block:: python
+    # Launch multiple of these at the same time to see the lock in action
+    import time
+    import fasteners
+
+    @fasteners.process_lock.interprocess_locked('tmp_lock_file')
```

Comment:
How about we use ``fasteners.interprocess_locked`` instead of having to use the full path here.

Also how about naming the lock "/tmp/tmp_lock_file" instead (as its a more valid path).